Sources_v3/Workers/Background/MissingEventsPublisher.swift

```diff
@@ -67,7 +67,7 @@ class MissingEventsPublisher<ExtraData: ExtraDataTypes>: Worker {
                 cids: watchedChannelIDs
             )
             
-            self?.apiClient.request(endpoint: endpoint) { [weak self] in
+            self?.apiClient.request(endpoint: endpoint) {
```

Comment:
You removed this because `self` is already `weak` or was there another reason?

Reply:
Exactly that reason